src/constants/extensions.ts

```diff
@@ -9,6 +9,15 @@ import {
   convertPeriodToSeconds,
 } from '~utils/extensions';
 
+import oneTransactionHero from '~images/assets/extensions/one-transaction-hero.png';
+import oneTransactionInterface from '~images/assets/extensions/one-transaction-interface.png';
```

Comment:
This image is cut off as opposed to the designs

![Screenshot from 2023-11-13 12-18-50](https://github.com/JoinColony/colonyCDapp/assets/1193222/d239d549-a731-4e9b-ab02-aa5d1c75132b)

vs.

![Screenshot from 2023-11-13 12-18-57](https://github.com/JoinColony/colonyCDapp/assets/1193222/a71d0aea-aec3-42b6-82b6-ae9af8787ae5)